arch/x86/kvm/pkvm/pkvm.c

```diff
@@ -37,6 +53,49 @@ static void pkvm_disable_virtualization_cpu(void)
 	kvm_x86_call(disable_virtualization_cpu)();
 }
 
+static int pkvm_vm_init(phys_addr_t host_kvm_pa, phys_addr_t pkvm_vm_pa)
+{
+	struct pkvm_vm *pkvm_vm;
+	struct kvm *shared_kvm;
+	size_t size;
+	u8 vm_type;
+	int ret;
+
+	ret = pkvm_host_share_hyp(host_kvm_pa, kvm_x86_call(vm_size));
```

Comment:
As we agreed in https://android-review.googlesource.com/c/kernel/common/+/3783093, kvm_x86_call(vm_size) seems hacky since vm_size is not a function, so let's just use kvm_x86_ops.vm_size.

Reply:
Yes. I will update.